### DISCUSSION TITLE
Anomaly detection integration

### DISCUSSION THREAD

**Discussion by gchumash** [Security-Onion-Solutions/securityonion#4343] [Unsupported Versions]
How do we integrate logs/detections from anomaly detection engines?

Any ideas?

**Comment by presianbg** [comment]
Hi, would yo be so kind to elaborate on your question?
You want to ingest logs/alerts from an anomaly detection system into SO or vice versa ?

**Reply by gchumash** [reply]
Thx, Yes, I want to be able ingest alerts from ADS into SO. Thanks
